### ISSUE TITLE
Pupil Player not starting with eye videos

### ISSUE THREAD

**Issue by blarkj** [pupil-labs/pupil#1020]
Hey,

I am using the eye tracker with 200 Hz binocular eye camera with Pupil version 1.2_7. I can record the videos with Pupil Capture but am unable to play them with Pupil Player (1.2_7). 

The eye videos are recorded at 400x400 res with120 Hz and scene video is recorded at 1280x720 res with 60 Hz. Yet all these videos don't play in the player.

Kindly help!

**Comment by papr** [comment]
Hey @blarkj 
No worries, by default the eye videos are not shown. You should load the `Vis Eye Video Overlay` plugin if you want to see the eye video in relation to the world window. Use a normal video player (e.g. VLC) if you want to inspect the eye videos only. Enable `Offline Pupil Detection` to run the pupil detection algorithm on the recorded eye videos.

**Comment by blarkj** [comment]
Hello @papr ,
Thank you for reverting. 
The Vis Eye Video Overlay causes Pupil Player to crash. However, the Player crashes ONLY if the eye videos are recorded at 120 Hz with 400x400 resolution.
I recorded the video at 200 Hz with 192x192 resolution and all the videos (eyes and scene) played fine in the Player.

I initially thought the eye videos (400x400 res at 120 Hz) must have somehow got corrupted but they work fine in VLC. 
I need the eye videos to be 400x400 in resolution and so I tried the experiment multiple times and even installed on Pupil Player (1.2.7) on multiple machines, only to find, that the Player crashes every single time.

Your input will be highly appreciated on this.
